### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by jonathanhar**
After applying these changes I successfully used the `transform` function of the keras based sparkML model.

### CODE REVIEW

File: elephas/spark_model.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 from .optimizers import SGD as default_optimizer
 
 from keras.models import model_from_yaml
-from keras import constraints
+from keras.constraints import Constraint
```
Comment (maxpumperla) [inline elephas/spark_model.py]: you never seem to use `Constraint`, am I missing something?

**Comment by maxpumperla** [comment]
@jonathanhar lgtm, just one question above!

File: elephas/spark_model.py (hunk 1 above)
Comment (jonathanhar) [reply elephas/spark_model.py]: This is just in case someone would like to use real constraints in the future (the import refers to the right place)

Comment (jonathanhar) [reply elephas/spark_model.py]: I see what happened, I'll remove it

Comment (maxpumperla) [reply elephas/spark_model.py]: ok, just let me know. I'll merge after

**Comment by maxpumperla** [comment]
@jonathanhar thanks again and congrats to your first PR! May there be more to come :)

**Pull request merged**